util/gh-pages/index.html

```diff
@@ -448,6 +448,12 @@ <h1>Clippy Lints</h1>
                                         None
                                     </label>
                                 </li>
+                                <li class="checkbox">
```

Comment:
I would maybe put this between "All" and "None", since it's the middle ground. Though this version looks a bit better, as the word length increases

Reply:
I can agree with you. The order "All", "Default", "None" is natural.